avc/launch/start_detector.launch

```diff
@@ -1,20 +1,20 @@
 <launch>
```

Comment:
We may want to delete this file because I think we always invoke the _standalone versions. @barulicm @chsahit thoughts?

Reply:
I'd rather we actually figured out why it wasn't working in this configuration. Having it under the same nodelet manager as the camera will allow for more efficient image passing.

Reply:
I'm making this a post-merge issue